pkg/execution/state/redis_state/lua/includes/enqueue_to_partition.lua

```diff
@@ -60,7 +60,8 @@ local function enqueue_to_partition(keyPartitionSet, partitionID, partitionItem,
 		--       we've forced a partition to have a delay.
 		--
 		--       Here, we do those checks.
-		if nowMS > existing.forceAtMS then
+
+		if nowMS == nil or nowMS == false or existing == false or existing == nil or existing.forceAtMS == nil or nowMS > tonumber(existing.forceAtMS) then
```

Comment:
We were hitting partitions without forceAtMS set. `tonumber` returns nil if the passed argument is nil or false, so we need to guard against that case.

<img width="394" alt="image" src="https://github.com/user-attachments/assets/a6d28b64-c834-4b1b-aa97-306ed82fc7fe">